README.md

```diff
@@ -75,68 +78,101 @@ Use SnowCLI to build a function or stored procedure, or create a streamlit if yo
+   Snowflake username: <username>
+   Snowflake password: <password>
+   ```
 
-Provide a name for your connection, your account, username, and password for Snowflake.
+#### Add credentials using a configuration file
 
-#### Add credentials manually on *nix systems
+Snowflake CLI lets you add connection definitions to a configuration file.
+A connection definition refers to a collection of connection parameters.
 
-If you do not want to install SnowSQL, you can add Snowflake account credentials manually:
+To add credentials in a configuration file:
 
-1. In your home directory, make `.snowsql` directory with a `config` file:
+1. In your home directory, create a :file:`~/.snowflake` directory:
 
-   `mkdir .snowsql`
-   `cd .snowsql`
-   `touch config`
+   ```
+   $ mkdir ~/.snowflake
+   ```
 
-2. Open the config file for editing:
+1. Create a :file:`config.toml` file in that directory:
 
-   `vi config`
+   ```
+   $ cd ~/.snowflake
+   $ touch config.toml
+   ```
 
-3. Add a new configuration for your Snowflake connection with SnowCLI. You must prefix the configuration with `connections.`.
+1. In a text editor, open the :file:`config.toml` file for editing, such as the following for the Linux vi editor:
 
-For example, to add a Snowflake account `myaccount` for a user profile `johndoe` and a password of `hunter2`, add the following:
+   ```
+   $ vi config.toml
+   ```
 
-```ini
-[connections.connection_name]
-accountname = myaccount
-username = jondoe
-password = hunter2
-```
+1. Add a new Snowflake connection definition. You must prefix the configuration with **connections**.
 
-#### Add credentials manually using Windows
+   For example, to add a Snowflake connection called **myconnection** with the credentials: account **myaccount**,
+   user profile **johndoe**, and password **hunter2**,
+   add the following lines to the configuration file:
 
-1. Create a `.snowsql` folder with a `config` file at the following path: `%USERPROFILE%\.snowsql\config`
-2. Add a new configuration for your Snowflake connection with SnowCLI. You must prefix the configuration with `connections.`.
+   ```
+   [connections.myconnection]
+   accountname = myaccount
+   username = jondoe
+   password = hunter2
+   ```
 
-### Using environment variables
+1. If desired, you can add more connections, as shown:
 
-Environment variables can only be replaced in connection parameters. Parameter values in config file have to start with `$SNOWCLI_` and will be searched as a name in environment variables, but without `$` at the beginning.
+   ```
+   [connections.myconnection-test]
+   accountname = myaccount
+   username = jondoe-test
+    password = hunter2
+   ```
 
-For example connection config:
+1. Save changes to the file.
 
-```ini
-[connections.test]
-account = $SNOWCLI_ACCOUNT
-user = $SNOWCLI_USER
-password = $SNOWCLI_PASSWORD
-```
 
-will be replaced with environment variables:
+#### Use environment variables for Snowflake credentials
```

Comment:
We decided to use solution from python connector and remove our. Their replacing works a bit differently, we have to provide list of possible environment variable names. It should be done in few days, so please hold off on that part for a while